### ISSUE TITLE
Can self-supervise method be used ?

### ISSUE THREAD

**Issue by tranxuantuyen** [VIPriors/vipriors-challenges-toolkit#69]
Hi, can I use the self-supervise method for pretraining (like masking following BERT-style) ?

I noticed that the webpage mentions "Unsupervised learning through visual priors, e.g., contrastive learning," but also states the rule of no pretraining.

**Comment by rjbruin** [comment]
Hi, you can use method that does not rely on other training data or on model weights that were trained using other data.
